src/incubator/TextField/types.ts

```diff
@@ -215,7 +215,7 @@ export type TextFieldProps = MarginModifiers &
   };
 
 export type InternalTextFieldProps = PropsWithChildren<
-  TextFieldProps & BaseComponentInjectedProps & ForwardRefInjectedProps
+  TextFieldProps & BaseComponentInjectedProps & ForwardRefInjectedProps & {placeholderColor: ColorType}
```

Comment:
Shouldn't we add the `placeholderColor` type to `TextFieldProps` with the rest of the props?

Reply:
Found the correct prop: `placeholderTextColor`, already there :)